### PR TITLE
Removed CoreBundle deprecations

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Removed
- Removed CoreBundle deprecations
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,12 +21,12 @@
     ],
     "require": {
         "php": "^7.1",
-        "sonata-project/admin-bundle": "^3.35",
-        "sonata-project/block-bundle": "^3.11",
+        "sonata-project/admin-bundle": "^3.45",
+        "sonata-project/block-bundle": "^3.14",
         "sonata-project/cache-bundle": "^2.1.7",
-        "sonata-project/core-bundle": "^3.9",
+        "sonata-project/core-bundle": "^3.14",
```
Comment (greg0ire) [inline composer.json]: Travis crashes because of the conflict with exporter, try maybe with `^3.13.7`, it does not have that conflict.

**Comment by core23** [comment]
Please review again @kunicmarko20 @greg0ire

**Comment by OskarStark** [comment]
Did you check the docs, too?

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**